static/js/showDetailTable.js

```diff
@@ -5,6 +5,118 @@
 
 // TODO, rename
```

Comment:
Please put these three global variables into a namespace, since the name is very general and they are not going to use outside the fucntion.